exonum-java-binding/launcher-plugin/README.md

```diff
@@ -0,0 +1,43 @@
+# Java Runtime Plugin for Exonum Launcher
+
+## Installation
+
+#### Requirements
+
+- [Python3](https://www.python.org/downloads/)
+- [`exonum-python-client`](https://github.com/exonum/exonum-python-client)
+- [`exonum-launcher`](https://github.com/exonum/exonum-launcher)
+
+```bash
+cd exonum-java-binding
+python3 -m pip install -e launcher-plugin
+```
+
+## Usage
+
+Add `plugins` session to configuration file of the Exonum Launcher:
+
+```yaml
+runtimes:
+  java: 1
+
+plugins:
+  runtime:
+    java: "exonum_java_runtime_plugin.JavaRuntimeSpecLoader"
+  artifact: {}
+```
+
+To load service artifact, provide a path to the service artifact, for example:
```

Comment:
not a path, but a file name — and please specify where the file shall be

Reply:
Renamed, added todo to move this entire section to ejb app tutorial. This readme is not intended to be used by our end-users.